src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -374,14 +379,15 @@ public void fireFileStarted(String fileName) {
      * Notify all listeners about the errors in a file.
      *
      * @param fileName the audited file
-     * @param errors the audit errors from the file
+     * @param fileProcessingResult file processing results object
      */
     @Override
-    public void fireErrors(String fileName, SortedSet<LocalizedMessage> errors) {
+    public void fireErrors(String fileName, FileProcessingResult fileProcessingResult) {
```

Comment:
@MEZk Should `fileName` be in `FileProcessingResult`?
It seems like they are fully conntected. `FileText` can also retrieve the `File` object, which can get it's name too.